src/main/java/it/pagopa/pn/service/desk/service/impl/ApiKeysServiceImpl.java

```diff
@@ -17,16 +20,24 @@ public class ApiKeysServiceImpl implements ApiKeysService {
 
     private final ApiKeysManagerClient apiKeysManagerClient;
 
+    private final PnAuditLogBuilder auditLogBuilder = new PnAuditLogBuilder();
```

Comment:
Dovremmo usare PnAuditLogBuilder come variabile di metodo invece che di instanza in quanto non è thread-safe

Reply:
modifica effettuata, su tutte le classi è stato fatto l'Autowired di un service creato per il build dell'audit log